doc/guide/privileges.xml

```diff
@@ -39,4 +39,38 @@ $ pkexec cockpit-bridge
     user to be able to escalate privileges in this way. However both Policy Kit and
     sudo may be configured to use other criteria.</para>
 
+  <section id="privileges-polkit">
+    <title>Customizing Polkit Privileges</title>
+
+    <para>Services like <ulink url="http://www.freedesktop.org/wiki/Software/systemd/">systemd</ulink>
+      and <ulink url="https://wiki.gnome.org/Projects/NetworkManager">NetworkManager</ulink> use
+      <ulink url="http://www.freedesktop.org/wiki/Software/polkit/">Policy Kit</ulink> to
```

Comment:
The legacy name is "PolicyKit", the current name is "polkit"; I've never seen "Policy Kit".